src/main/java/net/consensys/wittgenstein/protocols/ETHPoW.java

```diff
@@ -210,6 +219,7 @@ private void setHashPower() {
 
     private POWBlock inMining;
     private double threshold;
+    private List<POWBlock> uncles = new ArrayList<>();
```

Comment:
It seems strange to have this here: the uncle list should be created "on demand" (and there can be multiple strategies to select them)